### PR TITLE
WIP: Media/remove attachment too deep

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: assets/src/edit-story/app/media/utils/createResource.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * Internal dependencies
+ */
+import getTypeFromMime from './getTypeFromMime';
+
+/**
+ * Resource object.
+ *
+ * @todo: Try to remove posterId (poster should be enough) and videoId (src)
+ * should be enough.
+ * @todo: We should just get the non-formatted numeric length. Formatting can
```
Comment (swissspidy) [inline assets/src/edit-story/app/media/utils/createResource.js:27]: This is already possible, just not passed along by the `APIPRovider`. Check out #714 to see how it uses the non-formatted length.

File: assets/src/edit-story/app/media/utils/createResource.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * resource.
+ * @property {boolean} local Whether the resource has been already uploaded to
+ * the server.
+ */
+
+/**
+ * Creates a resource object.
+ *
+ * @param {Object} attachment WP Attachment object
+ * @return {Object} Resource object
```
Comment (swissspidy) [inline assets/src/edit-story/app/media/utils/createResource.js:54]: Actually use `Resource` from above?

File: assets/src/edit-story/app/api/apiProvider.js [hunk 3]
```diff
@@ -142,16 +145,16 @@ function APIProvider({ children }) {
...
-              posterId,
-              poster,
-              src,
-              oWidth,
-              oHeight,
+            }) => createResource({
               mimeType,
+              src,
+              width: oWidth,
+              height: oHeight,
```
Comment (obetomuniz) [inline assets/src/edit-story/app/api/apiProvider.js:152]: nit: `width` and  `height` can be passed directly.

**Comment by spacedmonkey** [comment]
@dvoytenko Can you provide some context, detail and issue for what this PR is for. Thanks.

**Comment by dvoytenko** [comment]
This was a test pull requests that turned into this: https://github.com/google/web-stories-wp/pull/908. But basically I just cleaned up the shapes of a few objects. E.g. removed "oWidth" and "oHeight" and such.

**Pull request closed without merging**